packages/runtime/container-runtime/src/test/gcHelpers.spec.ts

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { shouldDisableGcEnforcement } from "../garbageCollectionHelpers";
+
+describe("Garbage Collection Helpers Tests", () => {
+	describe("shouldDisableGcEnforcement", () => {
+		const testCases: {
+			persisted: number | undefined;
+			current: number | undefined;
+			expectedShouldDisableValue: boolean;
+		}[] = [
+			{
+				persisted: undefined,
+				current: undefined,
+				expectedShouldDisableValue: false,
```

Comment:
I would expect this to be true.

Reply:
Yeah I wrestled with this.  Keeping it simple for now, it will not hinder the immediate use case our partners have.

I think we may want persisted `undefined` value to be mapped to 0 first, then compare with the incoming current value from the option. See https://dev.azure.com/fluidframework/internal/_workitems/edit/3424 for more details.

Reply:
In the other thread you said:

> why not return false if persistedValue is undefined? Before applications start passing the GC option, this would be undefined and tombstone will be enabled right?

I don't want this change to disable tombstone until someone wants it disabled.

Reply:
Just doing some mental translating: Tombstone should be enabled until someone wants it disabled, right?

Reply:
Yes.  I think I'm confident enough now to make the change to say if input is undefined, always allow (don't disable)